src/pages/kb/open-source/admin-guide/env-vars-settings.md

```diff
@@ -65,6 +65,7 @@ The follow is a list of settings and what they control:
 | `REDASH_CORS_ACCESS_CONTROL_ALLOW_CREDENTIALS`  |                                                                                                                                                                                                                                                                    | false                                                                                                                                      |
 | `REDASH_CORS_ACCESS_CONTROL_REQUEST_METHOD`     |                                                                                                                                                                                                                                                                    | GET, POST, PUT                                                                                                                             |
 | `REDASH_CORS_ACCESS_CONTROL_ALLOW_HEADERS`      |                                                                                                                                                                                                                                                                    | Content-Type                                                                                                                               |
+| `REDASH_ENFORCE_CSRF`                           | Enforce CSRF token validation on API requests. **This is turned off by default to avoid breaking any existing deployments. but it is highly recommended to turn this toggle on to prevent CSRF attacks.**                                                          | false                                                                                                                                      |
```

Comment:
`... deployments. but is highly ...`

The full stop after "deployments" should be a comma instead. :wink:

Reply:
Good point, I changed. :)